console/src/components/Services/Settings/Actions.js

```diff
@@ -37,7 +37,9 @@ const getInconsistentObjectsQuery = {
 
 const reloadCacheQuery = {
   type: 'reload_metadata',
-  args: {},
+  args: {
+    reload_remote_schemas: true,
```

Comment:
@wawhal Perhaps this should only be set if there are one ore more remote schemas which are inconsistent? Otherwise, this can get quite needlessly heavy operation.

Reply:
@tirumaraiselvan @rakeshkky  can the server not make this check? I will have to make an extra network request to get this information because we fetch remote-schemas/event-triggers/actions only in their respective routes.

I was thinking that server might have access to the information about number of remote schemas so it'd be better done there.

I can do it if you absolutely cannot. Let me know.

Reply:
The server cannot by the API design itself. Without that arg, the API behaviour is to ignore remote schemas. And with that arg, the API behaviour is to not ignore remote schemas. There is no way to say "reload remote schemas if remote schemas are inconsistent".

I am not sure I understand the problem. The `get_inconsistent_metadata` API should tell you if there are inconsistent remote schemas right?

Reply:
I am saying, people might want to reload their metadata (including remote schemas) generally. So this must not be tied to the inconsistent objects.

I have done this now.

![Screenshot from 2020-03-22 21-53-58](https://user-images.githubusercontent.com/27274869/77254595-fda42800-6c87-11ea-91af-e95419e3b42c.png)

Similar for the inconsistent metadata page. Is this okay? They'll check the flag if they want to reload remote schemas as well. One more improvement we can do is, if there's an inconsistent remote schema, we can have the checkbox checked by default.

Reply:
Yep, this is perfect.